cmd/api-server/main.go

```diff
@@ -242,6 +243,17 @@ func main() {
 				log.DefaultLogger.Infow("minio is not available, using default logs storage", "error", err)
 			}
 		}
+
+		// Run DB migrations
+		dbMigrator, err := dbmigrator.NewDbMigrator(db, "__migrations", cfg.TestkubeConfigDir)
+		ui.ExitOnError("Loading MongoDB migrations", err)
```

Comment:
Should it be error or warning? Is it critical?

Reply:
I think that when MongoDB migrations will be a part of code, we should rather consider them as critical error. Each new part of code may depend on some migrations (i.e. if migration will rename `status` property name to `executionStatus` and that migration won't be ran, the API will not be working correctly)

Reply:
We can take the "warning" approach too (as MongoDB is rather flexible, so often lack of migrations will not break anything), but it's just less safe

Reply:
great